app/services/create_nomis_visit.rb

```diff
@@ -1,9 +1,11 @@
 class CreateNomisVisit
   ALREADY_BOOKED_IN_NOMIS = 'Duplicate post'.freeze
+  PVB_USER_ID_HEADER_FIELD = 'X-PVB-User-id'.freeze
```

Comment:
@StupidCodeFactory - any particular reason for using the custom header ('X-') convention? I don't think it's required

Reply:
good stop will change